resources/web/wwi/WebotsView.js

```diff
@@ -37,7 +39,7 @@ export default class WebotsView extends HTMLElement {
     this.#initialCallbackDone = true;
 
     this.toolbarCss = document.createElement('link');
-    this.toolbarCss.href = 'https://cyberbotics.com/wwi/R2023b/css/toolbar.css';
+    this.toolbarCss.href = 'https://cyberbotics.com/wwi/proto/css/toolbar.css';
```

Comment:
should this be changed prior to merge?

Reply:
It should be change when merging, if we change it know we will not be able to test